executor/analyze_col_v2.go

```diff
@@ -653,8 +660,11 @@ workLoop:
 						Value:   val,
 						Ordinal: j,
 					})
+					// tmp memory usage
+					deltaSize := val.MemUsage() + 4
```

Comment:
Could you add a comment on where 4 comes from?

Reply:
Updated